Packs/Thycotic/Integrations/Thycotic/Thycotic_description.md

```diff
@@ -0,0 +1,4 @@
+  ## Thycotic Secret Server
```

Comment:
in this file we usually explain the steps needed to configure the user to use the integraion in order for it to work properly. e.g:


To use Thycotic on Cortex XSOAR, ensure your user account has the foo role.
You can do that by Selecting the "account" tab -> "Permissions"  -> ...
These role is required to make API calls.

Reply:
@AndyNikolaev I have fixed the build and errors for you.
However, it seems the integration's description is still pending refactoring from your side.
Please improve it and add the data requested in previous comments, and we'll be good to go

Reply:
@AndyNikolaev It still seems like the description could use more details.
If you need more tips or a detailed explanation from me, just let me know.
You can see some examples in what @bakatzir wrote above